groovy/postInit/mod/OpenComputers.groovy

```diff
@@ -1183,16 +1167,6 @@ for (solder in soldering_alloys) {
             .EUt(120)
             .buildAndRegister();
 
-	mods.gregtech.assembler.recipeBuilder()
```

Comment:
This was intended for the sticky piston upgrade, I believe, as it has metaitem number 33 (and the recipe makes sense for it).

Reply:
You should probably add this back as a result, but for the right metaitem.

Reply:
I merged with your commits which includes meta 33, consider this fixed.